### PR TITLE
Features/switch

### PR DESCRIPTION
**Pull request by yinchyy**

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+sed -i -E '/devDependencies/,/}/{/"react"|"react-dom"/d}' package.json && yarn && rollup -c && sed -i -e '23a\' -e '    "react": "\^18.2.0"\,\n    "react-dom": "\^18.2.0"\,' package.json && yarn
```
Comment (pilotpirxie) [inline scripts/build.sh:1]: IMO split with \ per line to make it easier readable

File: src/Checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -12,6 +13,7 @@ export function Checkbox({
   return (
     <input
       type="checkbox"
+      className={cx({ [styles.checkbox]: true })}
```
Comment (pilotpirxie) [inline src/Checkbox/Checkbox.tsx:16]: hmm isn't this always true? so you can 
```
 className={styles.checkbox}
```

File: src/Switch/Switch.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+export type SwitchProps = {
+  /**
+   * ID of checkbox, also used for label
+   */
+  id:string;
+   /**
+   * Should switch be turned on by default?
+   */
+  checked?: boolean;
```
Comment (pilotpirxie) [inline src/Switch/Switch.tsx:13]: In checkbox you have ``defaultChecked`` make it consistent.

File: src/Switch/Switch.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  id,
+  checked = false,
+  ...props
+}: SwitchProps) {
+  const [isChecked, setIsChecked] = useState(checked);
+
+  return (
+    <div
+      className={cx({
+        [styles.switch]: true,
```
Comment (pilotpirxie) [inline src/Switch/Switch.tsx:29]: it's always true so you can
```
cx(styles.switch, {
[styles.outlinePrimary]: isChecked,
[styles.outlineSecondary]: !isChecked,
})
```

No need to pass true

File: src/Switch/switch.module.scss [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  position: relative;
+  border-radius: 6px;
+  & label{
+    display: block;
+    width: 14px;
+    height: 16px;
+    border-radius: 3px;
+    transition: all .3s ease;
+    cursor: pointer;
+    position: absolute;
```
Comment (pilotpirxie) [inline src/Switch/switch.module.scss:16]: is this position absolute necessary?

**Review by pilotpirxie — APPROVED** [review]

**Review by pilotpirxie — CHANGES_REQUESTED** [review]
^

**Comment by pilotpirxie** [comment]
@yinchyy

File: scripts/build.sh (hunk 1 above)
Comment (yinchyy) [reply scripts/build.sh]: resolved [e6d4049](https://github.com/pilotpirxie/react-toucan-ui/pull/8/commits/e6d4049f5c6515106dd7bb2f5c846be8f6b8f34a)

File: src/Checkbox/Checkbox.tsx (hunk 2 above)
Comment (yinchyy) [reply src/Checkbox/Checkbox.tsx]: resolved [c203084](https://github.com/pilotpirxie/react-toucan-ui/pull/8/commits/c20308454c438b19459f1dbe29cfc1dd8ae1c704)

File: src/Switch/Switch.tsx (hunk 3 above)
Comment (yinchyy) [reply src/Switch/Switch.tsx]: resolved [c203084](https://github.com/pilotpirxie/react-toucan-ui/pull/8/commits/c20308454c438b19459f1dbe29cfc1dd8ae1c704)

File: src/Switch/Switch.tsx (hunk 4 above)
Comment (yinchyy) [reply src/Switch/Switch.tsx]: resolved [c203084](https://github.com/pilotpirxie/react-toucan-ui/pull/8/commits/c20308454c438b19459f1dbe29cfc1dd8ae1c704)

File: src/Switch/switch.module.scss (hunk 5 above)
Comment (yinchyy) [reply src/Switch/switch.module.scss]: It isn't :smile: 
changed to transform: translate()
[dfde1d1](https://github.com/pilotpirxie/react-toucan-ui/pull/8/commits/dfde1d1e1399dba8a80e1411d1ff6433c0067c72)

**Review by pilotpirxie — APPROVED** [review]

**Pull request merged**